docs/core/tutorials/top-level-templates.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Template changes in .NET 6"
+description: The console app template now uses top-level statements. Understand what's changed and how to use existing learning materials with the new program.
+ms.date: 07/13/2021
+---
+# New templates generate top-level statements
+
+Starting with the .NET 6 Preview 7 SDK, the console app template generates the following code:
+
+```csharp
+Console.WriteLine("Hello, World!");
+// See https://aka.ms/new-console-template for more information
```

Comment:
[This URL](https://aka.ms/new-console-template) doesn't go anywhere meaningful yet, is that intentional?

Reply:
Yes. It will go to this article once published.  (This article will be live before preview 7 goes public)